src/nsls2api/cli/proposal.py

```diff
@@ -150,7 +148,7 @@ def view(
         error(f"Failed to retrieve proposal with {lookup_type} = {identifier}.")
         raise typer.Exit(code=1)
 
-    proposal_data = response.json().get("proposal", {})
+    proposal_data = response.json().get("proposal", None)
```

Comment:
This is fine, though does have the same behavior as `.get()` default behavior.
```suggestion
    proposal_data = response.json().get("proposal")
```

Certainly there is no harm in being explicit though.

Reply:
Yeah - I was just being explicit to avoid any confusion. 😄